README.md

```diff
@@ -58,7 +59,7 @@ curl -L https://github.com/openpubkey/opkssh/releases/latest/download/opkssh-osx
 To install on linux run:
 
 ```bash
-curl -L https://github.com/openpubkey/opkssh/releases/latest/download/opkssh-linux-amd64 -o opkssh; chmod +x opkssh
+curl -L https://github.com/openpubkey/opkssh/releases/latest/download/opkssh-linux-$(uname -p) -o opkssh; chmod +x opkssh
 ```
```

Comment:
On ubuntu and centos `uname -p` resolves to `x86_64` not `arm64` which breaks the install